server/routes/api.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const mongoose = require('mongoose');
 const Product = require('../models/product');
+const User = require('../models/user');
 
 const router = express.Router();
```

Comment:
Group similar routes together in their own file. The goes is to have high cohesion: https://medium.com/clarityhub/low-coupling-high-cohesion-3610e35ac4a6